common/src/main/java/io/druid/common/config/Log4jShutdown.java

```diff
@@ -28,11 +28,12 @@
 import javax.annotation.concurrent.GuardedBy;
 import java.util.Queue;
 import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class Log4jShutdown implements ShutdownCallbackRegistry, LifeCycle
 {
-  private static final long SHUTDOWN_WAIT_TIMEOUT = 60000;
+  private static final long SHUTDOWN_WAIT_TIMEOUT = TimeUnit.MINUTES.toMillis(1);
```

Comment:
This is nice, but any chance you could be convinced to also rename these variables to include the unit of time they represent? e.g. `SHUTDOWN_WAIT_TIMEOUT_MILLIS` instead fo `SHUTDOWN_WAIT_TIMEOUT`

Reply:
Hi, @clintropolis . Thanks for your comments. It's has been done.

Reply:
Oops, I didn't mean just this variable, I meant all of the ones that were changed. Sorry for the confusion!

Reply:
I just also realized it, yes, I will fix them all.

Reply:
Done. :sweat_smile: